src/Microsoft.AspNet.Mvc.Core/Internal/MvcCoreMvcOptionsSetup.cs

```diff
@@ -51,9 +51,9 @@ public static void ConfigureMvc(MvcOptions options, IHttpRequestStreamReaderFact
             options.OutputFormatters.Add(new StreamOutputFormatter());
 
             // Set up ValueProviders
+            options.ValueProviderFactories.Add(new FormValueProviderFactory());
```

Comment:
Looks like there is one - https://github.com/aspnet/Mvc/blob/dev/test/Microsoft.AspNet.Mvc.Test/MvcOptionsSetupTest.cs#L71 - shouldn't it fail?